src/vuestic-theme/vuestic-components/va-list/VaListLabel.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="va-list-label title title--info" :style="{'color': color}">
```

Comment:
`:style="{'color': color}"` > `:style="{'color': colorComputed}"` maybe? Should provide fallback colors this way.

Reply:
By the way, `.title--info` won't react to `info` theme color changes, as far as I know. Should remove this one and use `colorComputed` instead (and see my comment fo docs).

Reply:
fixed